src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -55,19 +58,26 @@
     private static final String TAG_ACHIEVEMENT_LIST       = "achievementlist";
     private static final String TAG_WORK                   = "work";
     private static final String TAG_MANUAL_HIRING          = "manualHiring";
+    private static final String TAG_WAYPOINT               = "waypoints";
+    private static final String TAG_BLOCK                  = "blockState";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "TAG_BLOCK" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1068)